compiler/noirc_evaluator/src/ssa/opt/normalize_value_ids.rs

```diff
@@ -76,13 +76,14 @@ impl Context {
             new_function.dfg.make_global(value.get_type().into_owned());
         }
 
-        let mut reachable_blocks = PostOrder::with_function(old_function).into_vec();
-        reachable_blocks.reverse();
-
+        let reachable_blocks = old_function.reachable_blocks().into_iter().collect::<Vec<_>>();
```

Comment:
Can `populate_blocks` accept an iterator to avoid `collect`ing into ` a `Vec` here?

Reply:
yeah I think it can I'll try that

Reply:
`populate_blocks` accepts an `impl IntoIterator` now